Common/include/toolboxes/C2DContainer.hpp

```diff
@@ -90,9 +90,9 @@ class AccessorImpl
    * runtime internal value that depend on the number of rows/columns.
    * What values need setting depends on the specialization as not all have
    * members for e.g. number of rows and cols (static size optimization).
-   * \note Aligned dynamic allocation is disabled for compilation on MAC.
+   * \note Aligned dynamic allocation is disabled for compilation on MAC and Windows.
    */
-#ifndef __APPLE__
+#if !defined __APPLE__ && !defined _WIN64 && !defined _MSC_VER
```

Comment:
Why not use `boost::alignment::aligned_alloc` from `boost/align/aligned_alloc.hpp`? It would require adding `align` to boost archive already distributed with tecio.

Reply:
Might as well make that and `#if LINUX` at this rate :)

Reply:
We can think of using boost at some point. At the moment I am a little bit hesitant as this would add another mandatory dependency and we get all kind of problems related to that (I have seen that in other projects). The boost files we have for tecio are stripped down and always shipped as the tecio library does not change frequently.

Reply:
Ok, MSVC actually has it but under different names: `_aligned_free` and `_aligned_malloc` from `malloc.h` https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/aligned-malloc?view=vs-2019
For Mac, this is from boost:
```c++
BOOST_CONSTEXPR inline bool
is_alignment(std::size_t value) BOOST_NOEXCEPT
{
    return (value > 0) && ((value & (value - 1)) == 0);
}

inline void*
aligned_alloc(std::size_t alignment, std::size_t size) BOOST_NOEXCEPT
{
    BOOST_ASSERT(detail::is_alignment(alignment));
    if (size == 0) {
        return 0;
    }
    if (alignment < sizeof(void*)) {
        alignment = sizeof(void*);
    }
    void* p;
    if (::posix_memalign(&p, alignment, size) != 0) {
        p = 0;
    }
    return p;
}
```
So no boost dependency is even needed.